### PR TITLE
Added langchain_version while calling discover API

### PR DESCRIPTION
**Pull request by dristysrivastava**

### CODE REVIEW

**Review by Raj725 — APPROVED** [review]

File: libs/community/langchain_community/chains/pebblo_retrieval/base.py [hunk 1]
```diff
@@ -326,6 +327,7 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            langchain_community_version=langchain_community.__version__,
```
Comment (srics) [inline libs/community/langchain_community/chains/pebblo_retrieval/base.py:330]: We should make this field framework neutral. Perhaps we can reuse `plugin_version`? The value it takes could be `langchain-community x.y.z`, `llamaindex x.y.z`, etc.

Comment (srics) [reply libs/community/langchain_community/chains/pebblo_retrieval/base.py:335]: @dristysrivastava We can keep `plugin_version` for backward compatibility, deprecate and remove it in future releases. However, given we will support a wide variety of client types (overtime) like `langchain`, `llamaindex`, `haystack`, `autogen` .. and ETL partners, it makes sense to keep this neutral.

Comment (dristysrivastava) [reply libs/community/langchain_community/chains/pebblo_retrieval/base.py:335]: Updated it to generic field, here as well as in pebblo.

File: libs/community/langchain_community/document_loaders/pebblo.py [hunk 2]
```diff
@@ -406,6 +408,10 @@ def _get_app_details(self) -> App:
             runtime=runtime,
             framework=framework,
             plugin_version=PLUGIN_VERSION,
+            client_version=Framework(
+                name="langchain_community_version",
+                version=langchain_community.__version__,
+            ),
```
Comment (Raj725) [inline libs/community/langchain_community/document_loaders/pebblo.py:414]: Is it necessary to send both `langchain` version in the `framework` field and `langchain_community` version in `client_version`? Do we need both?

File: libs/community/langchain_community/chains/pebblo_retrieval/base.py [hunk 3]
```diff
@@ -331,6 +333,10 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            client_version=Framework(
+                name="langchain_community_version",
```
Comment (srics) [inline libs/community/langchain_community/chains/pebblo_retrieval/base.py:337]: `name` should be just pkg name, i.e., `langchain_community`. Also, is there a way to autodetect the current pkg name and version using utils like `importlib.metadata`?

File: libs/community/langchain_community/document_loaders/pebblo.py (hunk 2 above)
Comment (dristysrivastava) [reply libs/community/langchain_community/document_loaders/pebblo.py:414]: Currently, it is being used in pebblo report and DAXA UI, to show the version, but eventually it should update with langchain_community_version.
Let @srics confirm whether we should update/remove this

Comment (dristysrivastava) [reply libs/community/langchain_community/document_loaders/pebblo.py:414]: https://github.com/dristysrivastava/langchain/blob/6c43012f509705015066cf26e52656106e41343a/libs/community/langchain_community/document_loaders/pebblo.py#L409

**Review by Raj725 — APPROVED** [review]

**Pull request merged**